src/components/dialog/faultEvent/FaultEventCreation.tsx

```diff
@@ -82,7 +83,7 @@ const FaultEventCreation = ({ useFormMethods, isRootEvent, eventValue, isEditedE
           defaultValue={defaultValue}
         />
 
-        {!selectedEvent && !isRootEvent && (
+        {!selectedEvent && !isRootEvent && eventTypeWatch !== EventType.EXTERNAL && (
```

Comment:
consider renaming variable

eventTypeWatch -- strange name convention, not sure what i should imagine behind that. Can we be more specific?

Reply:
this comes from `const eventTypeWatch = watch("eventType");` so to me it makes sense to leave it like this. Otherwise simply `eventType` would be enough.
(I realize I merged before resolving this, I will make the changes in next PR and reference the commit here if need be)

Reply:
Good point, thanks.